docs/readthedocs/source/doc/Nano/Howto/index.rst

```diff
@@ -16,6 +16,16 @@ PyTorch Lightning
 * `How to conduct BFloat16 Mixed Precision training in your PyTorch Lightning application <Training/PyTorchLightning/pytorch_lightning_training_bf16.html>`_
 * `How to accelerate a computer vision data processing pipeline <Training/PyTorchLightning/pytorch_lightning_cv_data_pipeline.html>`_
 
+PyTorch
+~~~~~~~~~~~~~~~~~~~~~~~~~
+* |use_nano_decorator_pytorch_training|_
+* |convert_pytorch_training_torchnano|_
+
+.. |convert_pytorch_training_torchnano| replace:: How to convert your PyTorch training loop to use ``TorchNano`` for acceleration
+.. _convert_pytorch_training_torchnano: Training/PyTorch/convert_pytorch_training_torchnano.html
+.. |use_nano_decorator_pytorch_training| replace:: How to wrap your PyTorch training loop with ``@nano`` decorator for acceleration
```

Comment:
May change to `How to accelerate your PyTorch training loop with ``@nano`` decorator`

Reply:
Thank you! have updated